tests/integration/conftest.py

```diff
@@ -54,17 +70,78 @@ async def unit_ip_list(get_unit_ip_list):
 
 
 @pytest_asyncio.fixture(scope="module")
-async def app(ops_test: pytest_operator.plugin.OpsTest, content_cache_image: str):
+async def nginx_integrator_app(ops_test: OpsTest):
```

Comment:
I know this is merged, do you think it would be worth adding a docstring?

Reply:
My bad, I forgot. I'll open a PR for that and flake8.

Reply:
https://github.com/canonical/content-cache-k8s-operator/pull/10